src/lsm/lsm_cursor.c

```diff
@@ -539,7 +539,7 @@ retry:	if (F_ISSET(clsm, WT_CLSM_MERGE)) {
 				clsm->chunks[ngood - 1]->switch_txn =
 				    chunk->switch_txn;
 				if (__wt_txn_visible_all(
-				    session, chunk->switch_txn))
+				    session, chunk->switch_txn, NULL))
```

Comment:
@agorrod, let's talk about whether LSM needs to track a timestamp when switching chunks.  I thought not when I was changing the code but it's worth discussing (and documenting) if that's correct.